pentaho-js/src/main/javascript/cdf-legacy/components/Pentaho.Analyzer.js

```diff
@@ -55,6 +56,12 @@
         // process params and update options
         $.map(this.parameters, function(k) {
             options[k[0]] = k.length == 3 ? k[2] : Dashboards.getParameterValue(k[1]);
+            if(typeof myself.dateFormats !== 'undefined' && myself.dateFormats[k[0]]){
```

Comment:
CDF uses JSDoc 3, e.g. [here](https://github.com/webdetails/cdf/blob/master/core-js/src/main/javascript/cdf/dashboard/Dashboard.parameters.js#L32), you can use it instead of line comments.